src/main/java/com/pubnub/api/enums/PNPushType.java

```diff
@@ -4,6 +4,6 @@
 
     APNS,
     MPNS,
-    GCM
-
+    GCM,
```

Comment:
Do we need to include `FCM` too?

Reply:
Yes and no. 

The best solution would be to drop GCM completely and use FCM instead. 

No matter GCM or FCM, in the end it's the same in regards to the SDK functionality, but the docs need to be updated. 

Also a permit is needed to do so.